server/datastore/mysql/software_test.go

```diff
@@ -3714,25 +3715,35 @@ func testListHostSoftware(t *testing.T, ds *Datastore) {
 
 	// add VPP apps, one for both no team and team, and two for no-team only.
 	va1, err := ds.InsertVPPAppWithTeam(ctx,
-		&fleet.VPPApp{VPPAppID: fleet.VPPAppID{AdamID: "adam_vpp_1", Platform: fleet.MacOSPlatform}, Name: "vpp1",
-			BundleIdentifier: "com.app.vpp1"}, nil)
+		&fleet.VPPApp{
+			VPPAppID: fleet.VPPAppID{AdamID: "adam_vpp_1", Platform: fleet.MacOSPlatform}, Name: "vpp1",
+			BundleIdentifier: "com.app.vpp1",
+		}, nil)
```

Comment:
Why are there so many formatting changes? Do you have max line length set?

Reply:
not entirely sure on that one, i reverted the formatting changes